recipe/meta.yaml

```diff
@@ -20,13 +20,13 @@ requirements:
     - python
     - setuptools
     - pies2overrides  # [py2k]
-    - enum34          # [py<34]
+    - enum34          # [py2k or (py3k and py<34)]
```

Comment:
This could be pointing to conda-smithy not setting up the environment well enough for selectors.